static/src/javascripts/projects/commercial/modules/prebid/slot-config.js

```diff
@@ -6,51 +6,41 @@ export const slots: PrebidSlot[] = [
     {
         key: 'top-above-nav',
         sizes: [[970, 250], [728, 90]],
-        labelAll: ['desktop'],
     },
     {
         key: 'top-above-nav',
         sizes: [[728, 90]],
-        labelAll: ['tablet'],
     },
     {
         key: 'top-above-nav',
         sizes: [[300, 250]],
-        labelAll: ['mobile'],
     },
     {
         key: 'right',
         sizes: [[300, 600], [300, 250]],
-        labelAny: ['mobile', 'tablet', 'desktop'],
     },
     {
         key: 'inline1',
         sizes: [[300, 250]],
-        labelAny: ['mobile', 'tablet', 'desktop'],
     },
     {
         key: 'inline',
         sizes: [[300, 600], [300, 250]],
-        labelAll: ['desktop', 'article'],
     },
     {
         key: 'inline',
         sizes: [[300, 250]],
-        labelAll: ['desktop', 'non-article'],
     },
     {
         key: 'inline',
         sizes: [[300, 250]],
-        labelAny: ['tablet', 'mobile'],
     },
     {
         key: 'mostpop',
         sizes: [[300, 250]],
-        labelAny: ['mobile', 'tablet', 'desktop'],
     },
     {
         key: 'comments',
         sizes: [[300, 250]],
-        labelAny: ['desktop'],
     },
 ];
```

Comment:
Will need to do these size restrictions in conditions as well.  So that `top-above-nav` asks for different sizes according to breakpoint etc.